gtest_parallel.py

```diff
@@ -861,7 +861,10 @@ def find_tests(binaries, additional_args, options, times):
     test_names = parse_test_names(test_binary, list_command, options.gtest_also_run_disabled_tests)
     for test_name in test_names:
       
-      test_command = command + ['--gtest_filter=' + test_name]
+      if options.test != '':
+        test_command = command + ['--test=' + test_name]
+      if options.gtest_filter != '':
+        test_command = command + ['--gtest_filter=' + test_name]
```

Comment:
Can both be passed in as arguments? Would that mean `gtest_filter` prevails as the option then?

Reply:
If both --test and --gtest_filter are passed to the test executable, the values are combined. The --gtest_filter can include negative patterns. That is the only way it can remove tests added with --test.

Reply:
That sounds like this logic is correct then, ending up using `--gtest_filter` when both are there.

Reply:
Oh. I didn't realize that the second if statement would overrides the first. That's not right. Also, it is possible to pass --test multiple times. Is that handled?

Reply:
Yes, `--test` and `--gtest_filter` could both be passed in and are combined to generate the list of tests (`--gtest_list_tests`). When we run the individual test commands here, we need to pass along one or the other. Maybe we could pass both, I'm not sure what the most robust solution is here.

Reply:
Okay. I didn't appreciate the context. I would think either option should work to run an individual test and that this change is not needed.